templates/org_mozilla_vrbrowser_derived/metrics_daily_v1/metadata.yaml

```diff
@@ -0,0 +1,9 @@
+friendly_name: VR Browser Metrics Daily
+description: >
+  A daily aggregate of metrics pings from each Firefox Reality client,
+  partitioned by day
+owners:
+  - ascholtz@mozilla.com
```

Comment:
not sure if that should be me?

Reply:
If you feel comfortable with being listed as owner, go for it. But feel free to mark `jklukas@mozilla.com` as the owner, or as an additional owner.

Reply:
good idea, I added you as owner